starter_code/views/index.hbs

```diff
@@ -1,2 +1,6 @@
-<h1>{{title}}</h1>
-<p>Welcome to {{title}}</p>
+<div class='homepage'>
+    <h1>Cinema <br> Ironhack</h1>
+    <form class='button' action="/movies" method="GET">
+        <button class="button1" type="submit">Check the movies!</button>
+    </form>
```

Comment:
Here you don't need the form
`<button><a href="/movies">Check the movies!</a></button>`